src/components/Content.tsx

```diff
@@ -27,7 +27,7 @@ class Content extends React.Component<{}, IState> {
 
         return (
             <div className="app-wrapper">
-                <Editor
+                <ToolsMenu
```

Comment:
spis bych udelal komponentu `Sidebar` a do ni hodil uz pak presnejsi popis nez `ToolsMenu`, za par mesicu si nevzpomenes co vubec ToolsMenu je, ale to necham na tobe